src/index.css

```diff
@@ -17,7 +17,22 @@ code {
 }
 
 .p-button:hover {
-  transform: scale(1.02);
+  transform: scale(1.015);
```

Comment:
Does the 0.05 difference matter ? IF not let's keep 1.02 as its rounded of.

Reply:
It does not matter much but it looks better than before.

Reply:
if it's not a big difference, I'd go with the one that looks best 👍 .

Reply:
I agree 👍